unified-scheduler-pool/src/lib.rs

```diff
@@ -1586,16 +1586,16 @@ impl<S: SpawnableScheduler<TH>, TH: TaskHandler> ThreadManager<S, TH> {
                             }
                         },
                         recv(handler_context.banking_packet_receiver) -> banking_packet => {
+                            let HandlerContext {banking_packet_handler, banking_stage_helper, ..} = &mut handler_context;
+                            let banking_stage_helper = banking_stage_helper.as_ref().unwrap();
+
                             // See solana_core::banking_stage::unified_scheduler module doc as to
                             // justification of this additional work in the handler thread.
                             let Ok(banking_packet) = banking_packet else {
                                 info!("disconnected banking_packet_receiver");
                                 break;
                             };
-                            (handler_context.banking_packet_handler)(
-                                handler_context.banking_stage_helper.as_ref().unwrap(),
-                                banking_packet
-                            );
+                            banking_packet_handler(banking_stage_helper, banking_packet);
```

Comment:
this is small prep clean up for upcoming pr... i'm piggybucking on this pr.

Reply:
functionally there is little difference in behavior, unless I'm missing something.

Only difference I see is that now if the `banking_stage_helper` were `None` (I believe it cannot be), we'd panic before breaking in above if instead of after the if.